src/Sylius/Bundle/CoreBundle/Command/Helper/EnsureDirectoryExistsAndIsWritable.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Command\Helper;
+
+use Sylius\Bundle\CoreBundle\Installer\Checker\CommandDirectoryChecker;
+use Symfony\Component\Console\Output\OutputInterface;
+
+trait EnsureDirectoryExistsAndIsWritable
```

Comment:
Same here, in this we event have the constructor parameters :) I really believe we can just refactor these traits into some proper services